libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/AlterSoupLongOperation.java

```diff
@@ -495,9 +495,9 @@ private void copyOldData() {
 			List<String> oldColumnsFts = new ArrayList<String>();
 			List<String> newColumnsFts = new ArrayList<String>();
 
-			// Adding docid column
```

Comment:
With FTS5, they don't alias the id column as docid anymore.
The good news is that using rowid works against fts4 and fts5 tables.